Jenkinsfile

```
@@ -67,11 +67,10 @@ pipeline {
       }
     }
 
-    stage('Start File server for Integration Tests'){
+    stage('Start File-Server for Performance and Integration Testing'){
```

There are no Jmeter performance tests being done.
```suggestion
    stage('Start File-Server for Integration Tests'){
```